src/Plugin/WebformElement/Os2formsPersonLookup.php

```diff
@@ -0,0 +1,123 @@
+ *
+ * @see \Drupal\address\Element\Address
+ */
+class Os2formsPersonLookup extends WebformCompositeBase {
+  /**
+   * {@inheritdoc}
+   */
+  protected function defineDefaultProperties() {
+    $properties = parent::defineDefaultProperties();
+
+    $composite_elements = $this->getCompositeElements();
+    foreach ($composite_elements as $composite_key => $composite_element) {
+      if (isset($properties[$composite_key . '__required'])) {
+        $properties[$composite_key . '__required'] = TRUE;
+      }
+    }
+    return $properties;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function prepareElementValidateCallbacks(array &$element, WebformSubmissionInterface $webform_submission = NULL) {
+    parent::prepareElementValidateCallbacks($element, $webform_submission);
+
+    $element['#element_validate'][] = [get_class($this), 'validatePerson'];
+  }
+
+  /**
+   * {@inheritdoc}
+   *
+   * @see \Drupal\address\Plugin\Field\FieldType\AddressItem::schema
+   */
+  public function initializeCompositeElements(array &$element) {
+    $element['#webform_composite_elements'] = [
+      'cpr_number' => [
+        '#title' => $this->t('CPR number'),
+        '#type' => 'textfield',
+        '#maxlength' => 255,
+        '#required' => TRUE,
+      ],
+      'name' => [
+        '#title' => $this->t('Name'),
+        '#type' => 'textfield',
+        '#maxlength' => 255,
+        '#required' => TRUE,
+      ],
+    ];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getTestValues(array $element, WebformInterface $webform, array $options = []) {
+    return [
+      [
+        'cpr_number' => '123456-7890',
+        'name' => 'John Smith',
+      ],
+    ];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function prepare(array &$element, WebformSubmissionInterface $webform_submission = NULL) {
+    parent::prepare($element, $webform_submission);
+  }
+
+  /**
+   * Form API callback. Make sure CPR is valid and belong provided name.
+   */
+  public static function validatePerson(array &$element, FormStateInterface $form_state, array &$completed_form) {
+    $values = $element['#value'];
+    $cpr_number = str_replace('-', '', $values['cpr_number']);
+    $os2web_datalookup_plugins = \Drupal::service('plugin.manager.os2web_datalookup');
+
+    /** @var \Drupal\os2web_datalookup\Plugin\os2web\DataLookup\ServiceplatformenCPR $servicePlatformentCprPlugin */
+    $servicePlatformentCprPlugin = $os2web_datalookup_plugins->createInstance('serviceplatformen_cpr');
+    $peronsData = $servicePlatformentCprPlugin->cprBasicInformation($cpr_number);
+
+    if (!$peronsData['status']) {
+      $error = isset($peronsData['text']) ? $peronsData['text'] : (isset($peronsData['error']) ? $peronsData['error'] : t('Can not verify CPR Number'));
+      \Drupal::logger('os2forms')->warning(t('os2forms_person_lookup - data lookup error: @error', ['@error' => $error]));
+      $form_state->setError($element['cpr_number'], t('CPR is not valid.'));
```

Comment:
You should `use StringTranslationTrait` in the class and use `$this->t` here (cf. <https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21StringTranslation%21StringTranslationTrait.php/trait/StringTranslationTrait/9>).

Reply:
It's not possible to use `$this` context in static functions